packages/wrangler/templates/checked-fetch.js

```diff
@@ -1,9 +1,7 @@
 const urls = new Set();
 
 export function checkedFetch(request, init) {
-	const url = new URL(
-		(typeof request === "string" ? new Request(request, init) : request).url
-	);
+	const url = new URL(new Request(request, init).url);
```

Comment:
Ah, I think I got bitten by this change just the other day. Something to do with the request body being consumed twice. I'll look into it, but this should maybe be reverted.

Reply:
Yes let's revert this. I can see this error using the reproduction in the linked issue.

https://github.com/cloudflare/wrangler2/issues/2562#issuecomment-1400855328

Reply:
Here: https://github.com/cloudflare/wrangler2/pull/2597

Reply:
sorry about the bug :cry: (thanks for the fix Pete :bow:)

Reply:
I never thought to test it on POST requests coming out of a Worker.
The reproduction required a `fetch()` call to a `Request` that has a body (but GET requests cannot have bodies).